src/views/virtualmachines/details/tabs/details/components/sections/UserList/hooks/useActiveUsersColumnsVm.ts

```diff
@@ -0,0 +1,46 @@
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  K8sResourceCommon,
+  TableColumn,
+  useActiveColumns,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { sortable } from '@patternfly/react-table';
+
+const useActiveUsersColumnsVm = (): { title: string; id: string }[] => {
+  const { t } = useKubevirtTranslation();
+
+  const columns: TableColumn<K8sResourceCommon>[] = [
+    {
+      title: t('User Name'),
+      id: 'userName',
+      transforms: [sortable],
+      sort: 'metadata.name',
+    },
+    {
+      title: t('Domain'),
+      id: 'domain',
+      transforms: [sortable],
+      sort: 'domain',
+    },
+    {
+      title: t('Time of login'),
+      id: 'loginTime',
+      transforms: [sortable],
+      sort: 'loginTime',
+    },
+    {
+      title: t('Elapsed time since login'),
+      id: 'elapsedTime',
+    },
+  ];
+
+  const [activeColumns] = useActiveColumns<K8sResourceCommon>({
```

Comment:
useActiveColumns hook is used when we want to hide on some cases the namespace column,
here it's not needed since we don't have a namespace column here, please remove this block.